hack/test-end-to-end.sh

```diff
@@ -134,49 +170,35 @@ echo "[INFO] Pushed centos7"
 
 # Process template and apply
 echo "[INFO] Submitting application template json for processing..."
-osc process -n test -f examples/sample-app/application-template-${BUILD_TYPE}build.json > "${CONFIG_FILE}"
+osc process -n test -f examples/sample-app/application-template-stibuild.json > "${STI_CONFIG_FILE}"
```

Comment:
Since you've started naming each namespace after build type (which is great), I'd also apply this pattern to sti builds as well.